samples/peripheral/802154_phy_test/include/internal/ctrl/ptt_uart.h

```diff
@@ -36,6 +36,7 @@
 #define UART_CMD_R_SW_VERSION_TEXT "custom rsoftwareversion"
 #define UART_CMD_CHANGE_MODE_TEXT "custom changemode"
 #define UART_CMD_L_GET_CCA_TEXT "custom lgetcca"
+#define UART_CMD_L_SET_CCA_TEXT "custom lsetcca"
```

Comment:
It looks you are smuggling one more feature in "Antenna Diversity support" commit. I would split it to a separate commit. If I understand correctly, `lsetcca` is functionally separated from Antenna Diversity.